docs/MO_DG/prepare_model/convert_model/Converting_Model.md

```diff
@@ -101,18 +101,24 @@ Framework-agnostic parameters:
                         Parameter -> ReverseInputChannels -> Mean/Scale apply -> the original body of the model.                        
   --log_level {CRITICAL,ERROR,WARN,WARNING,INFO,DEBUG,NOTSET}
                         Logger level
-  --input INPUT         Quoted list of comma-separated input nodes names with
-                        shapes, data types, and values for freezing. The shape
-                        and value are specified as space-separated lists. The
-                        data type of input node is specified in braces and can
-                        have one of the values: f64 (float64), f32 (float32),
-                        f16 (float16), i64 (int64), i32 (int32), u8 (uint8),
-                        boolean. For example, use the following format to set
-                        input port 0 of the node `node_name1` with the shape
-                        [3 4] as an input node and freeze output port 1 of the
-                        node `node_name2` with the value [20 15] of the int32
-                        type and shape [2]: "0:node_name1[3
-                        4],node_name2:1[2]{i32}->[20 15]".
+  --input INPUT         Quoted list of comma-separated input nodes names with shapes, 
+                        data types, and values for freezing. The shape and value are 
+                        specified as space-separated lists. The    data type of input 
+                        node is specified in braces and can have one of the values: 
+                        f64 (float64), f32 (float32), f16 (float16), i64 (int64), 
+                        i32 (int32), u8 (uint8), boolean (bool). Data type is optional. 
+                        If it's not specified explicitly then there are two options: 
+                        if input node is a Parameter data type is inherited from the 
```

Comment:
`,` is missed; `parameter`; `taken`:
if input node is a parameter, data type is taken from the ...

Reply:
Agreed and rephrased